get_staples.hh

```diff
@@ -9,10 +9,10 @@
 
 template<class T, class S> void get_staples(T &K, gaugeconfig<S> &U,
                                    vector<size_t> const x,
-                                   const size_t mu, const double xi=1.0) {
+                                   const size_t mu, const double xi=1.0, bool anisotrope=false) {
```

Comment:
do we really need the ```anisptrope``` flag. Isn't it implicit when ```xi=1.0```

Reply:
Carsten suggested this, because it is easier to check a boolean than it is to compare doubles. We could always use the anisotropic code, and use xi=1, this is already the default value,  I do not think the additional multiplication/division takes a lot of time. 
However, at the moment I am also using it for the filenames in measure, because it might be nice to select if xi is included in the configurationname or not. 
I could take out the boolean, but then we will have to discuss where to include xi in the confignames, and if it is necessary for all of them.

Reply:
Isn't ```xi``` parsed at the beginning of the program? Maybe you could set a global boolean variable for the output filenames immediately after the parsing.

Reply:
I actually want to output xi=1 into my filenames, but I think you don't, so some boolean parameter is needed anyways. 
Setting a global variable would still require a comparison of two doubles, if(xi==1.0) , and Carsten said comparing doubles could lead to problems.